src/Repository/CommitteeRepository.php

```diff
@@ -334,4 +334,18 @@ public function paginate(int $offset = 0, int $limit = SearchParametersFilter::D
 
         return new Paginator($query);
     }
+
+    public function countApprovedCommitteesForReferentManagedArea(Adherent $referent): int
+    {
+        return (int) $this->createQueryBuilder('c')
+            ->select('COUNT(c) AS count')
+            ->innerJoin('c.referentTags', 'tag')
+            ->where('c.status = :status')
+            ->andWhere('tag IN (:tags)')
+            ->setParameter('tags', $referent->getManagedArea()->getTags())
+            ->setParameter('status', Committee::APPROVED)
+            ->getQuery()
+            ->getSingleResult()['count']
```

Comment:
can you try `getSingleScalarResult` instead of `getSingleResult()['count']`? what is the type of value if you remove `(int)` cast?

Reply:
It's a string. `getSingleScalarResult ` also returns a string.